src/Generating/ComposableGenerator.cpp

```diff
@@ -155,27 +155,33 @@ void cComposableGenerator::GenerateBiomes(cChunkCoords a_ChunkCoords, cChunkDef:
 
 void cComposableGenerator::Generate(cChunkDesc & a_ChunkDesc)
 {
+	// Wrap a cChunkDesc::Shape into a structure so that we can make a unique_ptr out of it:
+	struct ShapeHelper
```

Comment:
The struct should have a comment about musl and a link to #3406 for the record

Reply:
I don't think so, it's about the generic principle of keeping stuff off the stack, not about a specific issue.